second-analysis-steps/simulation.md

```diff
@@ -23,16 +23,16 @@ and a collection of useful links can be found on the project's website: http://l
 
 {% callout "Setup" %}
 
-Before we continue, please set up a Gauss v49r17 `lb-dev`:
-* Set up Gauss: `lb-dev Gauss/v49r17`
-* `cd ./GaussDev_v49r17`
+Before we continue, please set up a Gauss v49r21 `lb-dev`:
```

Comment:
do we need this "lb-dev" in this sentence?

Reply:
Please set up a Gauss v49r21 `lb-dev`:
Maybe ->
Please set up Gauss v49r21 using `lb-dev`:
?
If we remove the reference to `lb-dev`, we should remove the entire line since it would then be redundant with the line underneath

Reply:
agree!

Reply:
Also, actually running this code works but produces warnings and info which should be ignored, maybe we should say that too

```
[covatilla] ~/ImpactKit $ lb-dev Gauss/v49r21
WARNING:root:Gauss v49r21 does not seem a CMake project

Successfully created the local project GaussDev_v49r21 for x86_64-slc6-gcc49-opt in .

To start working:

  > cd ./GaussDev_v49r21
  > git lb-use Gauss
  > git lb-checkout Gauss/vXrY MyPackage

then

  > make
  > make test

and optionally (CMake only)

  > make install

To build for another platform call

  > make platform=<platform id>

You can customize the configuration by editing the files 'build.conf' and 'CMakeLists.txt'
(see https://twiki.cern.ch/twiki/bin/view/LHCb/GaudiCMakeConfiguration for details).

```
All of this should be ignored